proxy/version.go

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) OpenFaaS Author(s) 2018. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package proxy
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"time"
+)
+
+//GetSystemInfo get system information from /system/info endpoint
+func GetSystemInfo(gateway string, tlsInsecure bool) (map[string]interface{}, error) {
+	infoEndPoint := gateway + "/system/info"
+	timeout := 5 * time.Second
+
+	client := MakeHTTPClient(&timeout, tlsInsecure)
+	req, err := http.NewRequest("GET", infoEndPoint, nil)
+	if err != nil {
+		return nil, fmt.Errorf("cannot connect to OpenFaaS on URL: %s", gateway)
```

Comment:
don't think this is a connection err, types of errors thrown from  `http.NewRequest` are invalid http method or invalid url